### DISCUSSION TITLE
Can I use without PPPoE? (Cloud Gateway, MagentaTV)

### DISCUSSION THREAD

**Discussion by airmaxx** [fabianishere/udm-iptv#360] [Q&A]
Hi,
I've seen many posts with eth instead of pppoe, yet the description of this tool says pppoe is mandatory. My setup is a Cloud Gateway behind a Fibre FritzBox. NAT is disabled on the UCG. Instead of pppoe I'm using eth4 in my configuration. The firewall rules for MagentaTV are also implemented. My setup is still not working.

192.168.178.53 is address of the Cloud Gatway
192.168.1.0/24 is the network for the clients of the Cloud Gateway

```
=== Configuration ===
WAN Interface: eth4
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
5: eth4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    inet 192.168.178.53/24 scope global dynamic eth4
       valid_lft 735950sec preferred_lft 735950sec
192.168.178.0/24 proto kernel scope link src 192.168.178.53 
=== Service Logs ===
Nov 21 16:27:48 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Nov 21 16:27:48 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Nov 21 16:27:48 Cloud-Gateway-Max systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Nov 21 16:27:48 Cloud-Gateway-Max systemd[1]: Started IPTV support for the UniFi Dream Machine.
Nov 21 16:27:48 Cloud-Gateway-Max udm-iptvd[806898]: NATing IPTV network ranges (if necessary)
Nov 21 16:27:48 Cloud-Gateway-Max udm-iptvd[806898]: Creating static routes (if necessary)
Nov 21 16:27:48 Cloud-Gateway-Max udm-iptvd[806898]: Monitoring interface eth4 for changes
Nov 21 16:27:48 Cloud-Gateway-Max udm-iptvd[806898]: Setting up IGMP Proxy
Nov 21 16:27:48 Cloud-Gateway-Max udm-iptvd[806898]: Using improxy...
Nov 21 16:27:48 Cloud-Gateway-Max udm-iptvd[806898]: Starting IGMP Proxy

ifconfig
eth4: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.178.53  netmask 255.255.255.0  broadcast 0.0.0.0
        inet6 fe80::eea:14ff:fe39:feb5  prefixlen 64  scopeid 0x20<link>
        ether 0c:ea:14:39:fe:b5  txqueuelen 1000  (Ethernet)
        RX packets 9221852  bytes 7152468565 (6.6 GiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 7882652  bytes 8187560054 (7.6 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
        device base 0x1000 
```

Any idea what still is missing?

**Comment by dmutsaers** [comment]
Where did you see that pppoe is mandatory? I can't find it. I think your problem is that the cloud gateway is located behind the Fritzbox and not connected to the NTU.

> The FTTP NTU (or any other type of modem) of your ISP must be connected to one of the WAN ports of your UniFi device.

**Comment by airmaxx** [comment]
You are right, I may have misinterpreted the instructions. As the configuration for KN with pppoe is listed under prerequisites, I may have mistakenly assumed it was mandatory.

The FritzBox routes all traffic via a static route to the UCW and is connected to the WAN port of the UCW. Also, thanks to a UniFi update, there is no double NAT. Isn't that sufficient?

**Comment by dmutsaers** [comment]
The typical configuration places the public ip addresses on the WAN interface of the UCG (Max/Ultra), not the Fritzbox (or other device) in front of the UCG (Max/Ultra). Both VLANs should be assigned to the WAN interface of the UCG (Max/Ultra). The device in front of the UCG (Max/Ultra) should be a modem, not a router. 

        Fiber
          |
    +----------+
    | FTTH NTU |
    +----------+
          |
      VLAN4 - IPTV
      VLAN6 - Internet
          |
      +--------+
      | Router |  - Ubiquiti UniFi device
      +--------+
          |
         LAN
          |
      +--------+
      | Switch |  - Ubiquiti UniFi Switch (Optional)
      +--------+
       |  |  |
       |  |  +-----------------------------+
       |  |                                |
       |  +-----------------+              |
       |                    |              |
+--------------+       +---------+      +-----+
| IPTV Decoder |       | WiFi AP |      | ... |
+--------------+       +---------+      +-----+
  - KPN IPTV
  - Netflix
  
Is this your issue?
[UCG-max default WAN interface incorrect #351](https://github.com/fabianishere/udm-iptv/issues/351)

**Reply by thomt44** [reply]
Get rid of the fritzbox. useless
